index.html

```diff
@@ -28,5 +33,129 @@ <h3 class="subtitle">I'm a Software Developer</h3>
+        <ul class="socials">
+            <li class="boxes">HTML</li>
+            <li class="boxes">Bootstrap</li>
+            <li class="boxes">Ruby</li>
+        </ul>
+        <button type="button" class="regButton">See Project</button>
+        </div>
+    <div class="secondItem">
+        <h2 class="bigText">Profesional Art Printing Data</h2>
+        <p class="smallText">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="socials">
+            <li class="boxes">HTML</li>
+            <li class="boxes">Bootstrap</li>
+            <li class="boxes">Ruby</li>
+        </ul>
+        <button type="button" class="regButton">See Project</button>
+        </div>
+    <div class="secondItem">
+        <h2 class="bigText">Profesional Art Printing Data</h2>
+        <p class="smallText">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="socials">
+            <li class="boxes">HTML</li>
+            <li class="boxes">Bootstrap</li>
+            <li class="boxes">Ruby</li>
+        </ul>
+        <button type="button" class="regButton">See Project</button>
+    </div>
+    <div class="secondItem">
+        <h2 class="bigText">Profesional Art Printing Data</h2>
+        <p class="smallText">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="socials">
+            <li class="boxes">HTML</li>
+            <li class="boxes">Bootstrap</li>
+            <li class="boxes">Ruby</li>
+        </ul>
+        <button type="button" class="regButton">See Project</button>
+    </div>
+</div>
+<!-- End of cards -->
+  
+</section>
+<!-- Third page -->
+<section class="aboutMe">
+    <h1 class="bioTitle">About Me</h1>
+    <p class="bio">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+    <button type="button" class="resume">Get My Resume</button>
+    <img class="whiteLine" src="css/whiteBorder.png" alt="white line">
+<!--     Lists  -->
+    <h2 class="listTitle rectangleIcon">Languages</h2>
+    <ul class="itemsList">
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+    </ul>
+  <h2 class="listTitle rhombusIcon">Frameworks</h2>
+  <ul class="itemsList">
+    <li>Bootstrap</li>
+    <li>Ruby on Rails</li>
+    <li>RSPec</li>
+    <li>Capybara</li>
+    <li>Selenium</li>
+  </ul>
+  <h2 class="listTitle circleIcon">Skills</h2>
+  <ul class="itemsList">
+  <li>Codekit</li>
+  <li>GitHub</li>
+  <li>Codepen</li>
+  <li>Gitlab</li>
+  <li>Terminal</li>
+</ul>
```

Comment:
- [ ] [OPTIONAL] you forgot the icon next to the title of every skill please always try to stick to the Figma design 

- [ ] Your Design 
![image](https://user-images.githubusercontent.com/61790587/164803928-f2e828fb-6aad-4d3d-bf65-a74bfbe6dce4.png)

- [ ] Figma design
![image](https://user-images.githubusercontent.com/61790587/164804092-f29baafd-37fb-4b71-9b4c-8042e571a92a.png)

Reply:
@Daf98 Cool then 👍